gulp-ng-config.js

```diff
@@ -30,10 +32,24 @@ function gulpNgConfig (moduleName, configuration) {
         jsonObj,
         wrapTemplate;
 
-    try {
-      jsonObj = JSON.parse(file.contents.toString('utf8'));
-    } catch (e) {
-      this.emit('error', new PluginError(PLUGIN_NAME, 'invaild JSON file provided'));
+    if (_.endsWith(file.path, 'yml') || _.endsWith(file.path, 'yaml')) {
+      configuration.parser = 'yml';
```

Comment:
Right now, there is no way to know file type before L30. So, I am not sure how we could achieve this.